pkg/network/files.go

```diff
@@ -0,0 +1,107 @@
+package network
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"sort"
+	"strings"
+
+	"github.com/containernetworking/cni/libcni"
+	"github.com/containernetworking/plugins/plugins/ipam/host-local/backend/allocator"
+	"github.com/pkg/errors"
+)
+
+// LoadCNIConfsFromDir loads all the CNI configurations from a dir
+func LoadCNIConfsFromDir(dir string) ([]*libcni.NetworkConfigList, error) {
```

Comment:
I feel like we might want to wrap this in a sync.Once so we don't repeatedly reread the configs

Reply:
?? lets talk on Monday